src/main/java/LacksDataBaseProject/AddAndRemoveUser.java

```diff
@@ -0,0 +1,48 @@
+package LacksDataBaseProject;
+
+import java.util.Scanner;
+
+public class AddAndRemoveUser {
+    static Scanner input = new Scanner(System.in);
+
+    public static void addUser(User user) {
```

Comment:
Ten User, którego tu masz w atrybucie, to zwróć uwagę, że on nic tutaj nie robi. Jak go wywalisz tu i w mainie w wywolaniu metody to wszystko zadziala tak jak dzialalo.
1. User moze tu byc po to, zeby pola sie do niego odwolywaly, czyli zamiast String userName mogloby byc user.userName i wtedy jak bys tak wszędzie przerobil to bys nie musial tam na dole robic newUser tylko dodalbys od razu do listy add(user). Ja tak mam zrobiony Lack jak tworze, ale szczerze nie wiem czy to jest dobrze - moze tak jak ty masz jest lepiej... musimy Piotrka o to zapytać.
2. User moze byc w atrybucie po to zeby go zweryfikowac czy jest Adminem - i jesli jest to moze robic addUser, a jesli nie to nie i wydaje mi sie ze o to Piotrkowi chodzilo ze tak akurat mamy zrobic, wiec to bym dorobil...
3. No chyba ze ma byc 2 Userów w argumencie przekazywanych :)